pkg/util/fuzzystrmatch/leven_test.go

```diff
@@ -206,3 +206,139 @@ func TestLevenshteinDistanceWithCost(t *testing.T) {
 		}
 	}
 }
+
+func TestLevenshteinDistanceWithNegativeCost(t *testing.T) {
```

Comment:
it appears that this test does not run anything.

Reply:
Yes. We don't need this test as the logic for checking string lengths and int checks are in `builtins.go`.  

Removed the test.